### PR TITLE
Improve content sql ddl data types workout

### PR DESCRIPTION
**Pull request by kapnobatai137**
1. SQL Numeric Data Types (1)

- changed all syntax to ANSI Standard
- exact numeric
  - DECIMAL, SMALLINT, INT, BIGINT
- switched revision and practice question
- adjusted questions to fit new content
- wrote footnote for ANSI Standard
- added ANSI Standard resource link
- removed revision question

2. SQL Numeric Data Types (2)

- exact numeric
  - DECIMAL, SMALLINT, INT, BIGINT
- changed all syntax to ANSI Standard
- adjusted questions to fit new content
- removed revision question

3. SQL Numeric Data Types (3)

- booleans
-date & time
  - DATE, TIME, TIMESTAMP
- changed all syntax to ANSI Standard
- adjusted questions to fit new content

4. SQL String Data Types

- added ANSI Standard resource link
- changed data types to ANSI Standard
  - CHAR, VARCHAR, CLOB
- removed mention of bytes as it was confusing for beginners

The `SQL Text Data Type` and `Add Support for other character sets` insights have been archived because they contained information that was implementation specific. They will be added in a future insight that contains information regarding differences between ANSI and implementations.

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/sql/ddl/data-types/sql-numeric-data-types.md?ref=941274a1aee400a4f7e63fc500949d372b5b95fb
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/sql/ddl/data-types/sql-string-data-types.md?ref=941274a1aee400a4f7e63fc500949d372b5b95fb
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/sql/ddl/data-types/sql-text-data-type.md?ref=941274a1aee400a4f7e63fc500949d372b5b95fb

**Comment by enki-hq** [comment]
Rule insight:footnotes failed for files: 
  - sql/ddl/data-types/sql-numeric-data-types-2.md
  - sql/ddl/data-types/sql-numeric-data-types-3.md

**Comment by enki-hq** [comment]
Rule insight:footnotes failed for files: 
  - sql/ddl/data-types/sql-numeric-data-types-2.md
  - sql/ddl/data-types/sql-numeric-data-types-3.md

**Comment by kapnobatai137** [comment]
Split `Encoding & Collation` into 2 insights:
- Encoding
  - removed revision question
- Collation
  - removed practice question

File: sql/ddl/data-types/sql-string-data-types.md [hunk 1]
```diff
@@ -32,59 +36,69 @@ aspects:
 ---
 ## Content
 
-In terms of *string data types*, there are less options to be used. However, there are no discrepancies regarding naming conventions between PostgreSQL, MySQL and MSSQL. The three main types are:
-- `char(n)`
-- `varchar(n)`
-- `text`
+In terms of *string data types*, there are fewer options that can used with the three main types being:
```
Comment (nem035) [inline sql/ddl/data-types/sql-string-data-types.md]: Missing a "be"

> there are fewer options that can **be** used with

**Pull request merged**